dashboard/app/templates.html

```diff
@@ -361,6 +361,7 @@ <h1><a href="/{{$.Namespace}}">syzbot</a></h1>
 	{{end}}
 
 	{{if not .Reported}}[report pending]<br>{{end}}
+	{{if .IsAdmin}}{{deleteBisectionLink .JobKey "❌ delete this bisection result"}}{{end}}
```

Comment:
If we will have more such functions, we will need to add more such code, extend pkg/html with dashboard-specific logic (does not belong there) and pull more internal data into UI entities.
I think a slightly more scalable way to do this would be:
 - provide a link in the UI entity (empty if not available)
 - use `optlink` function to render it

I am also thinking if the handler can be reused. E.g. a common "admin action" handler with ?action parameter and action-specific args.

Reply:
Turns out we already have handleAdmin?action handler

Reply:
Done. I don't love the optlink mechanic, as it makes it less obvious when and why the link might not be populated, but it's better than stuff IsAdmin into uiJob for sure.